sdk/storage/storage-file-share/src/generated/src/models/index.ts

```diff
@@ -169,6 +169,10 @@ export interface ShareProperties {
   lastModified: Date;
   etag: string;
   quota: number;
+  provisionedIops?: number;
```

Comment:
i assume `Iops`/`MBps` is well known, and naming here is consistent with other languages?